### DISCUSSION TITLE
Popups for data tables?

### DISCUSSION THREAD

**Discussion by mma1504** [twilio-labs/paste#3169] [General] [answered]
I'm working on the error recommendation for data grids & tables, and for data grids my guidance is basically don't cram the whole error message into a given row. Either make the row selectable (so you can show a side sheet, or go to a details page, or whatever), but wondering if this kind of popup implementation is also possible/accessible/advisable?

https://www.lightningdesignsystem.com/components/data-tables/?variant=inline-edit#Edited-cell-with-row-level-error-(unsaved)

**Comment by libertycarlin** [comment, marked as answer]
The paste team and @mma1504 discussed this topic in slack.

The summary of this discussion was:

- immediacy of the delivery would trump length of the message. For example, as a user what am I trying to do?
It might be more important to get a long error at the moment than make me go to a new view.
- We can implement what SLDS does.
